docker-compose/ethereum-devnet.yml

```diff
@@ -20,4 +22,15 @@ services:
       test: ["CMD-SHELL", "curl -f http://localhost:8545 -X POST -H 'Content-Type: application/json' --data '{\"jsonrpc\":\"2.0\",\"method\":\"eth_blockNumber\",\"params\":[],\"id\":1}' || exit 1"]
       start_period: 5s
       interval: 5s
-      retries: 5
\ No newline at end of file
+      retries: 5
+
+  ethereum-setup:
```

Comment:
So it looks like `ethereum-setup` alone could be the testnet. It is running the foundry image.
Which `ethereum-devnet` also runs, only without the setup.

I think you can just have one container doing the setup and then running the testnet